src/components/StayedInDays.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
This is not really a component, just a function. So you don't really need to import React here